infor-design/enterprise-ng
# Misc - Add types for donut and datagrid

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Adds missing types for donut and datagrid. Also bumps the NG version

**Steps to Test**:
- go to http://localhost:4200/ids-enterprise-ng-demo/donut and hover the center label
- go to http://localhost:4200/ids-enterprise-ng-demo/datagrid-standard-formatter and see the radio buttons are there.

**Related github/jira issue (required)**:
#5384
#5302
